src/components/Button/index.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Button, { ButtonProps } from '@mui/material/Button';
```

Comment:
```suggestion
import { Button, ButtonProps, styled } from '@mui/material';
```

Move style definitions to `styles.ts`.